lib/rubocop/cop/sorbet/sigils/enforce_sigil_order.rb

```diff
@@ -49,6 +51,13 @@ def autocorrect(_node)
             tokens.each_with_index do |token, index|
               corrector.replace(token.pos, expected[index].text)
             end
+
+            # Remove blank lines between the magic comments
+            lines = Set.new
+            tokens.each { |token| lines << token.line }
+            each_extra_empty_line(lines.sort) do |range|
+              corrector.remove(range)
+            end
```

Comment:
The change below gives me the same output. Basically, we iterate on the whole range from the smallest to the largest line number and skip the line numbers that we have sigils on. Similarly, we skip lines that are not really empty and remove all the rest.

I feel like it is much simpler to reason about.

```suggestion
            lines = tokens.map(&:line).to_set
            (lines.min...lines.max).each do |line|
              next if lines.include?(line)
              next unless processed_source[line - 1].empty?
              corrector.remove(source_range(processed_source.buffer, line, 0))
            end
```

Reply:
Hmm I think it's definitely easier to reason about. Although it's less performant right. `include?` call would make it `O(n^2)`?

Not that performance really matters since it's just a rubocop autocorrect.

Reply:
But `lines` is a `Set` so the include check should be `O(1)`.

Reply:
Ah right. Pretty much same then other than iterating over all of the lines. Changing it now, thanks.